galleries/examples/specialty_plots/advanced_hillshading.py

```diff
@@ -43,11 +43,11 @@ def avoid_outliers():
     ls = LightSource(315, 45)
     fig, (ax1, ax2) = plt.subplots(ncols=2, figsize=(8, 4.5))
 
-    rgb = ls.shade(z, plt.cm.copper)
+    rgb = ls.shade(z, plt.colormaps["copper"])
```

Comment:
```suggestion
    rgb = ls.shade(z, cmap="copper")
```

Reply:
Hi @timhoffm after doing this suggested change I am getting an error,
```
 python sample.py
Traceback (most recent call last):
  File "/home/kaustubh/opensource/matplotlib/matplotlib2/matplotlib/sample.py", line 74, in <module>
    avoid_outliers()
  File "/home/kaustubh/opensource/matplotlib/matplotlib2/matplotlib/sample.py", line 46, in avoid_outliers
    rgb = ls.shade(z, cmap="copper")
          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kaustubh/opensource/matplotlib/matplotlib2/matplotlib/lib/matplotlib/colors.py", line 3481, in shade
    rgb0 = cmap(norm(data))
           ^^^^^^^^^^^^^^^^
TypeError: 'str' object is not callable
```

Reply:
Sorry, you are right. This must be a Colormap instance here.